assets/dev/js/editor/editor-base.js

```diff
@@ -1077,6 +1087,8 @@ export default class EditorBase extends Marionette.Application {
 			this.generateResponsiveControlsForElements();
 		}
 
+		this.elementsManager = new ElementsManager();
+
 		this.initComponents();
```

Comment:
Move to 
```suggestion
		this.initComponents();
```

Reply:
Handled https://github.com/elementor/elementor/pull/18424